samples/ChangefeedMigrationSample/Program.cs

```diff
@@ -68,26 +80,19 @@ public Task CloseAsync(ChangeFeedObserverContext context, ChangeFeedObserverClos
 
         public Task ProcessChangesAsync(ChangeFeedObserverContext context, IReadOnlyList<Document> docs)
         {
-            DocumentCollectionInfo destCollectionLocation = new DocumentCollectionInfo
-            {
-                Uri = new Uri("httpsURI"),
-                MasterKey = "PrimaryKey",
-                DatabaseName = "DestDBname",
-                CollectionName = "DestCollName"
-            };
-
-            DocumentClient newClient = new DocumentClient(destCollectionLocation.Uri, destCollectionLocation.MasterKey);
+            
+            DocumentClient newClient = new DocumentClient(this.destCollInfo.Uri, this.destCollInfo.MasterKey);
```

Comment:
Just like you did with destCollInfo, the newClient should be created outside ProcessChangesAsync. That way you are not recreating it on each trigger.